src/extension/windows/wpfPlatform.ts

```diff
@@ -31,52 +31,56 @@ export class WpfPlatform extends WindowsPlatform {
             },
         };
 
-        return TelemetryHelper.generate("WpfPlatform.runApp", extProps, () => {
-            const env = this.getEnvArgument();
+        return ReactNativeProjectHelper.getReactNativeVersionFromProjectPackage(this.runOptions.projectRoot)
+            .then(version => {
+                TelemetryHelper.addReactNativeVersionToEventProperties(version, extProps);
+                return TelemetryHelper.generate("WpfPlatform.runApp", extProps, () => {
+                    const env = this.getEnvArgument();
 
-            if (enableDebug) {
-                this.runArguments.push("--proxy");
-            }
-
-            return ReactNativeProjectHelper.getReactNativeVersion(this.runOptions.projectRoot)
-                .then(version => {
-                    if (!semver.gt(version, WpfPlatform.WPF_SUPPORTED)) {
-                        throw new Error(localize("DebuggingWPFPlatformIsNotSupportedForThisRNWinVersion", "Debugging WPF platform is not supported for this react-native-windows version({0})", version));
+                    if (enableDebug) {
+                        this.runArguments.push("--proxy");
                     }
 
-                    if (!semver.valid(version) /*Custom RN implementations should support this flag*/ || semver.gte(version, WpfPlatform.NO_PACKAGER_VERSION)) {
-                        this.runArguments.push("--no-packager");
-                    }
+                    return ReactNativeProjectHelper.getReactNativeVersion(this.runOptions.projectRoot)
```

Comment:
Why we get the version twice?

Reply:
I removed redundant getting of React Native version.